doc/sphinx/addendum/sprop.rst

```diff
@@ -13,21 +13,18 @@ known as strict propositions) as described in
 :cite:`Gilbert:POPL2019`.
 
 Using :math:`\SProp` may be prevented by passing ``-disallow-sprop``
-to the |Coq| program or using :flag:`Allow StrictProp`.
+to the |Coq| program or by turning the :flag:`Allow StrictProp` flag
+off.
 
 .. flag:: Allow StrictProp
    :name: Allow StrictProp
 
-   Allows using :math:`\SProp` when set and forbids it when unset. The
-   initial value depends on whether you used the command line
-   ``-disallow-sprop`` and ``-allow-sprop``.
+   Using :math:`\SProp` is allowed only when this flag is on (this is
+   the default).  It may also be globally turned off with the
+   command-line flag ``-disallow-sprop``.
```

Comment:
```suggestion
   Enables or disables the use of :math:`\SProp`.  It is enabled by default.
   The command-line flag ``-disallow-sprop`` disables :math:`\SProp` at startup.
```

I assumed you can override -disallow-sprop by setting the flag.

Should that be :math: or just ``` ` ... ` ```?  I guess the whole chapter is that way.